### DISCUSSION TITLE
Delete all previous markers from Node Playground

### DISCUSSION THREAD

**Discussion by vencivat** [foxglove/community#54] [Q&A] [answered]
Hi,

I publish MarkerArray messages from Node Playground. But my id are not stable, they change a lot.
As a result I get left overs markers. I see that there is DELETEALL=3 definition for marker action,
is this supported in Foxglove ? 

Also I see a lot of browser windows freezes. Is this a known issue? When this happens there is no debug info what is the cause
![image](https://user-images.githubusercontent.com/11574215/180263624-eb4e4db8-35ae-4fb7-b0a2-5a9f3dba67a6.png)

**Comment by jtbandes** [comment, marked as answer]
Yes, the action=3 is supported. We are not aware of any freezing issues. Is it possible that your Node Playground code has an infinite loop? Or is your computer under heavy load from running many other tasks?

**Reply by vencivat** [reply]
Thanks,
I will try desktop version and another computer to see if some antivirus is misbehaving
